brambling/static/brambling/sass/modules/_text.sass

```diff
@@ -70,6 +70,10 @@ h1, .h1
 
 //// Buttons
 
+// There was a weird line height thing going on with buttons.
```

Comment:
AFAICT this just increases the line height of buttons slightly, but this was a @nmorduch change - pinging in case she wants to add anything.